### PR TITLE
msys2-launcher: Switch to release v1.0

### PR DESCRIPTION
**Pull request by elieux**
Is the `conflicts` and `replaces` correct?

### CODE REVIEW

**Comment by lazka** [comment]
lgtm (I think conflicts is not needed because there already is one from the other package, but it shouldn't hurt)

**Comment by Biswa96** [comment]
`PREFIX=/opt/bin/...` ??

File: msys2-launcher/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+pkgdesc="Helper for launching MSYS2 shells"
+arch=('x86_64' 'i686')
+license=('MIT')
+groups=('base')
+backup=({mingw32,mingw64,msys2}.ini)
+conflicts=("${_realname}-git")
+replaces=("${_realname}-git")
+depends=("mintty>=1~2.2.1")
+makedepends=('mingw-w64-cross-gcc')
+source=("https://github.com/msys2/msys2-launcher/archive/v1.0.tar.gz")
```
Comment (Alexpux) [inline msys2-launcher/PKGBUILD:16]: Use pkgver instead hardcoding version in source URL

Comment (elieux) [reply msys2-launcher/PKGBUILD]: Thanks. Fixed.

**Comment by elieux** [comment]
> `PREFIX=/opt/bin/...` ??

It's probably a bad name for the variable. The idea is to have a simple way to cross-compile.

**Pull request merged**